src/utils/date.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Return day from the date
+ *
+ * @param {string} date || @param {Date object} date
+ * @return {string}
+ * @example
+ * getDateWeekday('2022-05-13') // 'Fri'
+ */
```

Comment:
I think that the comments are not necessary to understand the code and can be removed. 🚀 
That you already thought about how to convert the data is really cool, even though you could also have thought about that later and first work with data that matches the outcome of these functions. ;)